forward_script/src/QueryBuilder.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+
+namespace Wikimedia\ForwardScript;
+
+class QueryBuilder {
+
+	public function getQuery( $info, $pageName, $id = '', $coordinates = [] ) {
+		$category = preg_replace( '/^(:?Category|Kategorie):/', '', $info->category );
+		$query = array_merge( array_filter( $coordinates ), [ 'categories' => $category ] );
+		if ( $id && empty( $info->duplicate_ids ) && empty( $info->id_not_found ) ) {
```

Comment:
True, the `id_not_found` is either set to true are not present at all. I like the changed condition a bit better but it is just my opinion. What do you think about this?